db/product/product.go

```diff
@@ -6,22 +6,21 @@ import (
 	"github.com/BurdockBH/food-delivery-rest-service/db"
 	"github.com/BurdockBH/food-delivery-rest-service/viewmodels"
 	"log"
-	"time"
 )
 
-func CreateProduct(product *viewmodels.Product) error {
-	query := "CALL CreateProduct(?, ?, ?, ?, ? ,?, ?)"
+func CreateProduct(product *viewmodels.Product, email string) error {
```

Comment:
Are you validating the email format anywhere?

Reply:
comment above